samples/algorithms/chsh-game/Program.qs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.CHSHGame {
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Random;
+
+    /// # Summary
+    /// The number of times the classical and quantum strategies have won the CHSH game.
+    ///
+    /// # Named Items
+    /// ## ClassicalWins
+    /// The number of times the classical strategy won.
+    /// ## QuantumWins
+    /// The number of times the quantum strategy won.
+    internal newtype Score = (
+        ClassicalWins : Int,
+        QuantumWins : Int
+    );
+
+    /// # Summary
+    /// Compares the success rates of the classical and quantum CHSH game strategies across 10,000 trials.
+    @EntryPoint()
+    operation CompareStrategies() : Unit {
+        let trials = 10000;
+        let scores = DrawMany(PlayGame, trials, ());
+        let total = Fold(PlusScore, Score(0, 0), scores);
```

Comment:
I really love this way of writing out how to run lots of trials, but it may be unfamiliar to new Q# users such that adding some comments about what `DrawMany` and `Fold` are doing here could be helpful.

Reply:
I added comments for these lines. Do they help?